README.rst

```diff
@@ -73,6 +73,22 @@ Some Python HTTP clients that you can use to retrieve HTML include `requests`_,
 .. _urllib.request module: https://docs.python.org/3/library/urllib.request.html
 
 
+Content removal
+---------------
+
+This source code repository includes test HTML files from the public web that are used to quality-check the recipe web scrapers.  These test files are not included in the [published releases](https://pypi.org/project/recipe-scrapers/) of the library.
+
+If you would prefer for any of them to be removed, we recommend choosing one of the following processes:
+
+  * You may follow GitHub's [content removal request processes](https://docs.github.com/en/site-policy/content-removal-policies/submitting-content-removal-requests).
+
+  * You may [contact us](https://github.com/hhursev/recipe-scrapers/blob/6a6a9144c2be5ce299ce3805ea9344e34afadaca/pyproject.toml#L12-L14) (the software maintainers of this library) privately by email.
+
+  * You may [create a public bugreport for us](https://github.com/hhursev/recipe-scrapers/issues/new).  Please **do not** include your email address in your bugreport, because GitHub issues are public, and doing so could expose your email address to spammers.
```

Comment:
A [GitHub issue template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) for this would be helpful.

Reply:
(drafted in 2c6f762c21aec3f388cb659053fcce767410abd0)